modules/common/math/mpc_osqp_test.cc

```diff
@@ -0,0 +1,64 @@
+/******************************************************************************
+ * Copyright 2019 The Apollo Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *****************************************************************************/
+
+#include "modules/common/math/mpc_osqp.h"
+
+#include "cyber/common/log.h"
+#include "gtest/gtest.h"
+
+namespace apollo {
+namespace common {
+namespace math {
+
+TEST(MPCOSQPSolverTest, MPCOSQP) {
+  const int states = 4;
+  int controls = 2;
+  const int horizon = 2;
+  //   const double eps = 0.01;
+  const int max_iter = 100;
+
+  Eigen::MatrixXd A(states, states);
+  A << 1, 0, 1, 0, 0, 1, 0, 1, 0, 0, 1, 0, 0, 0, 0, 1;
+
+  Eigen::MatrixXd B(states, controls);
+  B << 0, 1, 0, 0, 1, 0, 0, 1;
+
+  Eigen::MatrixXd Q(states, states);
+  Q << 1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0;
+
+  Eigen::MatrixXd R(controls, controls);
+  R << 1, 0, 0, 1;
+
+  Eigen::MatrixXd lower_bound(controls, 1);
+  lower_bound << -10, -10;
+
+  Eigen::MatrixXd upper_bound(controls, 1);
+  upper_bound << 10, 10;
+
+  Eigen::MatrixXd initial_state(states, 1);
+  initial_state << 20, 0, 0, 0;
+  std::vector<double> control_cmd(2, 0);
+  ADEBUG << "IN TEST initialize";
+  MpcOsqp mpc_osqp_solver(A, B, Q, R, lower_bound, upper_bound, initial_state,
+                          max_iter, horizon);
+  ADEBUG << "IN TEST SOLVE";
+  mpc_osqp_solver.MpcOsqpSolver(&control_cmd);
```

Comment:
Is it better to call the api 'solve', or 'get_results'?

Reply:
Fixed. Thank you.